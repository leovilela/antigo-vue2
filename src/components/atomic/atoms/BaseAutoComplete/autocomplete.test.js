import { shallowMount } from "@vue/test-utils";
import AutoComplete from "../BaseAutoComplete/autocomplete";

const factory = () => shallowMount(AutoComplete, {
  propsData: {
    list: [],
  },
  stubs: ["Base-Input"],
});

describe("AutoComplete", () => {
  test("mounts properly", () => {
    const wrapper = factory();
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test("renders properly", () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
