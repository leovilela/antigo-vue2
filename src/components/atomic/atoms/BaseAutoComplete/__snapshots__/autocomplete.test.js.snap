// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AutoComplete renders properly 1`] = `
<form autocomplete="off">
  <div class="autocomplete" style="width: auto;"><label for=""></label>
    <base-input-stub type="text" value=""></base-input-stub>
  </div>
  <!---->
</form>
`;
